apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx

```diff
@@ -103,7 +104,7 @@ const EditMusicLevelData: React.FunctionComponent<EditMusicLevelDataProps> = ({
                 labelText="Selected Artist Pack"
                 name="packId"
                 size="s"
-                items={restrictedPacks}
+                items={[{value: 'none', text: '(none)'}, ...restrictedPacks]}
```

Comment:
Out of curiosity, how does this end up as no value for `packId` in the level data?

Reply:
Funnily enough I already had the code to handle this in the `onChange` below - if the value is "none", we'll set the value to `undefined`. I think I originally did have this "none" option but then must have removed it for some reason.